sdk/communication/Azure.Communication.Common/README.md

```diff
@@ -69,10 +69,14 @@ that a valid token is acquired prior to executing the request.
 Optionally, you can enable proactive token refreshing where a fresh token will be acquired as soon as the
 previous token approaches expiry. Using this method, your requests are less likely to be blocked to acquire a fresh token:
 
+Optionally, you can provide a time span before token expiry that `tokenRefresher` will be called if `refreshProactively` is true. For example, setting it to 5min means that 5min before the cached token expires, the proactive refresh will request a new token.
```

Comment:
```suggestion
Optionally, you can adjust the time span before token expiry that will trigger the proactive refreshing of the token. For example, if the proactive refreshing is enabled ('refreshProactively' is true), setting the time to a value equal to 5 minutes means that 5 minutes before the cached token expires, the proactive refresh will request a new token by calling the 'tokenRefresher' callback.
```

Reply:
@AikoBB have you intentionally not committed this one?

Reply:
guess I missed this one